operator-framework-core/src/main/java/io/javaoperatorsdk/operator/processing/event/source/informer/InformerConfiguration.java

```diff
@@ -0,0 +1,62 @@
+package io.javaoperatorsdk.operator.processing.event.source.informer;
+
+import java.util.Collections;
+import java.util.Objects;
+import java.util.Set;
+
+import io.fabric8.kubernetes.api.model.HasMetadata;
+import io.javaoperatorsdk.operator.api.config.ConfigurationService;
+import io.javaoperatorsdk.operator.api.config.DefaultResourceConfiguration;
+import io.javaoperatorsdk.operator.processing.event.ResourceID;
+import io.javaoperatorsdk.operator.processing.event.source.AssociatedSecondaryResourceIdentifier;
+import io.javaoperatorsdk.operator.processing.event.source.PrimaryResourcesRetriever;
+
+public class InformerConfiguration<R extends HasMetadata, P extends HasMetadata>
+    extends DefaultResourceConfiguration<R> {
+
+  private final PrimaryResourcesRetriever<R> secondaryToPrimaryResourcesIdSet;
+  private final AssociatedSecondaryResourceIdentifier<P> associatedWith;
```

Comment:
This should not abbe `associatedDependentResource`?

Reply:
Not sure what you mean…

Reply:
It's about naming, but I;m probably wrong. So I guess it's already this way to some extent, but:
1. The resources that are managed by dependent resources we can call  "dependent resources"
2.  And the secondary resource that is the resource managed by te controller by not the dependent resources mecahnism directly?

Or should not be rather managed resource and dependent resources?

Reply:
The naming is indeed not great… we should probably settle on using `dependent` or `secondary` everywhere.